### PR TITLE
Scalar market buy

### PR DESCRIPTION
**Pull request by KadenZipfel**
**Notes:** 
- This should only be reviewed and merged after https://github.com/protofire/omen-exchange/pull/1306 has been reviewed and merge (because this PR is branched off the other).

### CODE REVIEW

**Comment by pimato** [comment]
Awesome work!
A few comments:

can we reduce the vertical border to just one 1px width here:
Bildschirmfoto 2020-11-20 um 06 35 29

should:
Bildschirmfoto 2020-11-20 um 06 35 49

**Figma**:
[Design Specification](https://www.figma.com/file/R056tSYMjvfwaFidEBItYy/Widgets-V2?node-id=1209%3A141)

**Comment by pimato** [comment]
There is no border between the navigation button and the position form, currently:
Bildschirmfoto 2020-11-20 um 06 37 00

should:
Bildschirmfoto 2020-11-20 um 06 37 39

**Figma:**
[Design Specification](https://www.figma.com/file/R056tSYMjvfwaFidEBItYy/Widgets-V2?node-id=1209%3A141)

**Comment by pimato** [comment]
can we connect the tooltip to the dot, instead of heaving it float with the mouse:
Bildschirmfoto 2020-11-20 um 06 40 55

It should be horizontal aligned centered to the dot slider with a 12px margin-top:
Bildschirmfoto 2020-11-20 um 06 42 44

**Figma**:
[Design Specification](https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=66%3A4803)

**Comment by KadenZipfel** [comment]
> can we connect the tooltip to the dot, instead of heaving it float with the mouse:

I _really_ wanted to make this work but there's just no way to get the tooltip to follow an element :( I'm considering making a pull request to add that feature to the tooltip library

**Review by Mi-Lan — COMMENTED** [review]
Good job ! I found small thing...
Screenshot 2020-11-24 at 16 13 04
there should be + next to positive outcome...

**Review by Mi-Lan — COMMENTED** [review]
when i click max it calcualtes it but doesen't populate the field...
Screenshot 2020-11-24 at 16 27 28

File: app/src/theme/overrides/react_tooltip_styles_override.tsx [hunk 1]
```diff
@@ -67,4 +67,25 @@ export const ReactTooltipStylesOverride = css`
...
+      letter-spacing: 0.1px;
+      text-align: left;
+      background-color: ${theme.colors.mainBodyBackground};
+      opacity: 1;
+      border-radius: ${theme.borders.commonBorderRadius};
+      box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.05);
+      border: 1px solid ${theme.borders.tooltip};
+      padding: 5px 8px;
+    }
+  }
```
Comment (Mi-Lan) [inline app/src/theme/overrides/react_tooltip_styles_override.tsx:90]: @KadenZipfel you already probably though about it..but could this css not be repeated somehow seems that it's 90% similar to customMarketTolltip above it...

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
see comments

File: app/src/theme/overrides/react_tooltip_styles_override.tsx (hunk 1 above)
Comment (KadenZipfel) [reply app/src/theme/overrides/react_tooltip_styles_override.tsx:90]: Yeah, good call! Got lazy on this one

**Review by Mi-Lan — COMMENTED** [review]
@KadenZipfel what about those styles for the tooltips?

**Comment by KadenZipfel** [comment]
> @KadenZipfel what about those styles for the tooltips?

Whoops... Fixed now

**Review by Mi-Lan — APPROVED** [review]

**Comment by pimato** [comment]
I am not sure if the creation of a scalar market should work on this PR but I just tried to create one and it failed:
Bildschirmfoto 2020-12-07 um 11 29 26

**Review by pimato — CHANGES_REQUESTED** [review]
see comment

**Review by hexyls — APPROVED** [review]
👍

**Comment by pimato** [comment]
I am fine with merging for now but there are a few UI improvements to be made which we can introduce later on! Gonna create those issues once buy/sell and pool are merged into one PR!

**Review by pimato — APPROVED** [review]

**Pull request merged**